### PR TITLE
AO3-6583 Updated the robots.txt to block chatGPT.

### PR DESCRIPTION
**Pull request by JonathanS223**
# Pull Request Checklist

* [X] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [X] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [X] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [X] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [X] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue
[https://otwarchive.atlassian.net/browse/AO3-6583](https://otwarchive.atlassian.net/browse/AO3-6583)

## Purpose

This improves the robots.txt to block chatGPT from scraping AO3. I have also gone through the googlebot wild cards and they appear to all still be valid to keep Googlebot from scanning those. 

## Testing Instructions

This is a robots.txt file. We can use [https://www.websiteplanet.com/webtools/robots-txt/](https://www.websiteplanet.com/webtools/robots-txt/) to test the syntax. 

## References

NA

## Credit

Ivedonestranger (He/him)

### CODE REVIEW

File: public/robots.public.txt [hunk 1]
```diff
@@ -36,3 +34,6 @@ Disallow: /
 
 User-agent: Slurp
 Crawl-delay: 30
+
```
Comment (sarken) [inline public/robots.public.txt:37]: Based on [this discussion](https://github.com/otwcode/otwarchive/pull/4411#discussion_r1044351129), I think this should come before the user agent with the `crawl-delay`.

File: public/robots.public.txt [hunk 2]
```diff
@@ -23,7 +22,6 @@ Disallow: /*?*query=
 Disallow: /*?*sort_
 Disallow: /*?*selected_tags
 Disallow: /*?*selected_pseuds
-Disallow: /*?*use_openid
```
Comment (sarken) [inline public/robots.public.txt:26]: Would you mind also removing `Disallow: /*?*selected_pseuds` and `Disallow: /people?*show=`? I unfortunately didn't get the issue updated to mention those before you submitted your pull request, but I can update it now.

**Review by sarken — CHANGES_REQUESTED** [review]

File: public/robots.public.txt (hunk 2 above)
Comment (JonathanS223) [reply public/robots.public.txt:26]: I'd be happy too. Sorry for the fast response. I saw a way of blocking chatGPT and wanted to get this taken care of. lol

**Review by sarken — APPROVED** [review]
Thank you!

**Pull request merged**